Drift/Models/Embed.swift

```diff
@@ -91,7 +106,9 @@ struct Embed: Mappable {
         timeZoneString          <- map["configuration.theme.timezone"]
         backgroundColorString   <- map["configuration.theme.backgroundColor"]
         openHours               <- map["configuration.theme.openHours"]
-
+        userListModeRaw         <- map["configuration.theme.userListMode"]
```

Comment:
Ain’t no real here, we can use EnumTransform to go directly to the enum and ditch the raw value and the custom getter

Reply:
*realm